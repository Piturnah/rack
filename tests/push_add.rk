// Intended for testing mos_6502-nesulator target

20 22 +

// 42 should end up in accumulator