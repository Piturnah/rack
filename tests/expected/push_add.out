----STDOUT----

----STDERR----
[ERROR] No entry point `main` found.
